File: API/ContainerNinja.Contracts/DTO/CompletedOrderProductDTO.cs

namespace ContainerNinja.Contracts.DTO
{
    public class CompletedOrderProductDTO
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public long? WalmartId { get; set; }
        public string? WalmartItemResponse { get; set; }
        public string? WalmartSearchResponse { get; set; }
        public string? WalmartError { get; set; }
        public int CompletedOrderId { get; set; }

        public ProductDTO? Product { get; set; }
    }
}

File: Client/src/app/models/CompletedOrderProductDTO.ts
import { WalmartProductDTO } from "./WalmartProductDTO";

export class CompletedOrderProductDTO {
  id?: number = undefined;
  name?: string = undefined;
  walmartId?: number | undefined = undefined;
  walmartItemResponse?: string | undefined = undefined;
  walmartSearchResponse?: string | undefined = undefined;
  walmartError?: string | undefined = undefined;
  completedOrderId?: number = undefined;
  walmartProduct?: WalmartProductDTO | undefined = undefined;
}
